NetCoreRestApi/WebAPI/Controllers/ProductsController.cs

```diff
@@ -11,24 +11,24 @@ namespace WebAPI.Controllers
     [ApiController]
     public class ProductsController : ControllerBase
     {
-        private IProductConverter _productConverter;
+        private IConverter<ProductDTO, ProductModel> _converter;
         private IProductManager _productManager;
 
-        public ProductsController(IProductConverter productConverter, IProductManager productManager)
+        public ProductsController(IConverter<ProductDTO, ProductModel>  converter, IProductManager productManager)
         {
-            _productConverter = productConverter;
+            _converter = converter;
             _productManager = productManager;
         }
 
         [HttpGet]
-        public ActionResult<IEnumerable<ProductDTO>> Get()
+        public ActionResult<IQueryable<ProductDTO>> Get()
```

Comment:
use IActionResult interface instead so it should be testable

Reply:
I will try to do that

Reply:
done